addons/fetchmail/models/fetchmail.py

```diff
@@ -109,7 +109,7 @@ def connect(self):
                 connection = IMAP4_SSL(self.server, int(self.port))
             else:
                 connection = IMAP4(self.server, int(self.port))
-            connection.login(self.user, self.password)
```

Comment:
Comment about commit message:

I think it would be more readable if you specified the various steps using bullet points.
Something like:
> To enable the GMail support, users will go through these steps:
> - Go on the incoming / outgoing mail servers forms
> - Tick the GMail support box
> - They will be redirected to the GMail OAuth process to accept permissions
> ...